articles/cosmos-db/nosql/tutorial-dotnet-web-app.md

```diff
@@ -202,6 +202,21 @@ First, you'll create a database and container in the existing API for NoSQL acco
 
 Now, you'll create a new ASP.NET web application using a sample project template. You'll then explore the source code and run the sample to get acquainted with the application before adding Azure Cosmos DB connectivity using the Azure SDK for .NET.
 
+```output
+Before you start ensure Nuget.org is listed as a package source 
+"nuget source" command should prompt a similar output:
+
+C:\Users\username_here\nuget_tool>nuget source
+Registered Sources:
+  1.  nuget.org [Enabled]
+      https://api.nuget.org/v3/index.json
+  2.  Microsoft Visual Studio Offline Packages [Enabled]
+      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
+
+If https://api.nuget.org/v3/index.json is not listed as a source then add it with:
+nuget source add -Name "nuget.org" -Source "https://api.nuget.org/v3/index.json"
+```
```

Comment:
```suggestion
> [!IMPORTANT]
> This tutorial transparently pulls packages from [NuGet](https://nuget.org). You can use the [`dotnet nuget list source
`](/dotnet/core/tools/dotnet-nuget-list-source#examples) to verify your package sources. If you do not have NuGet as a package source, use [`dotnet nuget add source`](/dotnet/core/tools/dotnet-nuget-add-source#examples) to install the site as a source.
```

It's not considered a best practice for us to document how to perform a task in two different places. If the .NET CLI or NuGet team changes how to check package sources, this article will be out of date. Instead, we use a Note/Tip/Important callout and direct people to the appropriate documentation.

This revision implements our best practices.